src/docs/cookbook/animation/opacity-animation.md

```diff
@@ -144,7 +134,7 @@ AnimatedOpacity(
 );
 ```
 
-## Interactive example
+## Ví dụ tương tác
```

Comment:
Not sure whether we should use "ví dụ" or "ví dụ tương tác"

Reply:
Updated "Ví dụ tương tác" to "Ví dụ".